app.py

```diff
@@ -0,0 +1,14 @@
+# Adds logic to serve static html for running within huggingface
+from flask import send_from_directory, request
+from main import app
+
+@app.errorhandler(404)
+def page_not_found(e):
+    path = request.path.lstrip('/')
+    if path == '':
+       path = 'index.html'
+    print(path)
```

Comment:
I think we can skip this line. I think it was accidentally left in when working on serving static content.
```suggestion
```
Also could we move app.py to the `andromeda` directory with the rest of the python code?